### PR TITLE
Preparations for zcbor

### PR DESCRIPTION
**Pull request by VeijoPesonen**
Both content formats - plain CBOR and SenML CBOR - are kept disabled because my plan is to enable the formats implicitly from the engine side.

### CODE REVIEW

**Comment by VeijoPesonen** [comment]
@jheiskan81 Please review(s/review/merge/).

File: samples/nrf9160/lwm2m_client/prj.conf [hunk 1]
```diff
@@ -124,14 +124,6 @@ CONFIG_DATE_TIME_UPDATE_INTERVAL_SECONDS=86400
 
 CONFIG_LWM2M_VERSION_1_1=y
 CONFIG_LWM2M_PORTFOLIO_OBJ_SUPPORT=y
-CONFIG_BASE64=y
```
Comment (jheiskan81) [inline samples/nrf9160/lwm2m_client/prj.conf:127]: Why disable SenML Json?

**Review by jheiskan81 — APPROVED** [review]
Looks good only I was wondering disable senml json totally.

File: samples/nrf9160/lwm2m_client/prj.conf (hunk 1 above)
Comment (VeijoPesonen) [reply samples/nrf9160/lwm2m_client/prj.conf:127]: I assumed you don't need this branch yourself anymore but I can put it back if you need it.

File: samples/nrf9160/lwm2m_client/prj.conf [hunk 2]
```diff
@@ -127,11 +127,6 @@ CONFIG_LWM2M_PORTFOLIO_OBJ_SUPPORT=y
 CONFIG_BASE64=y
 CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
 CONFIG_LWM2M_RW_JSON_SUPPORT=y
-CONFIG_LWM2M_RW_CBOR_SUPPORT=y
 CONFIG_LWM2M_SECONDS_TO_UPDATE_EARLY=8
 CONFIG_LWM2M_COAP_MAX_MSG_SIZE=1024
 CONFIG_LWM2M_COMPOSITE_PATH_LIST_SIZE=10
-
-# Workaround for a missing cddl-gen Kconfig flag
```
Comment (jheiskan81) [inline samples/nrf9160/lwm2m_client/prj.conf:135]: can you revert commit which add this in this PR. So it not cause any conflict if we cherry pick things to main branch

**Pull request merged**